aten/src/ATen/native/cpu/UpSampleKernel.cpp

```diff
@@ -1287,6 +1287,25 @@ struct HelperInterpCubic : public HelperInterpBase {
 #undef a
   }
 
+  template<typename scalar_t>
+  static inline scalar_t aa_filter_075(scalar_t x) {
+    // https://en.wikipedia.org/wiki/Bicubic_interpolation#Bicubic_convolution_algorithm
+    // In this code we are using alternative a=-0.75 definition that matches
+    // get_cubic_upsample_coefficients implementation, as used by float path
+    // TODO: merge both implementations, the only difference is the value of a.
+    constexpr scalar_t a = -0.75;
+    if (x < 0.0) {
+      x = -x;
+    }
+    if (x < 1.0) {
+      return cubic_convolution1(x, a);
+    }
+    if (x < 2.0) {
+      return cubic_convolution2(x, a);
+    }
+    return 0.0;
```

Comment:
Please add a small comment explaining what is going on there
```suggestion
    if (x >= 2.0) {
      return 0.0;
    }
    if (x >= 1.0) {
      return cubic_convolution2(x, a);
    }
    return cubic_convolution1(x, a);
```

Reply:
Thanks @malfet 

We'll probably submit another PR  to unify this function with the one above (`aa_filter`) as noted in the `TODO` comment. For now I'd rather keep it as-is so that it's structurally the same as `aa_filter`. It will make unification easier.